src/ImageGallery.jsx

```diff
@@ -652,16 +654,17 @@ export default class ImageGallery extends React.Component {
     const CENTER = 'center';
     const RIGHT = 'right';
 
-    switch (index) {
-      case (currentIndex - 1):
-        alignment = ` ${LEFT}`;
-        break;
-      case (currentIndex):
-        alignment = ` ${CENTER}`;
-        break;
-      case (currentIndex + 1):
-        alignment = ` ${RIGHT}`;
-        break;
+    if (index === currentIndex - 1) {
```

Comment:
so should i think of index as previous index?

Reply:
No - `index` here refers to the position in the item list.